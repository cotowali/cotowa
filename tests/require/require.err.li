require 'ns.mod.li' { invalid_key: '' }
