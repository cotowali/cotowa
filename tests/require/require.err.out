error: require.err.li 1,23: invalid key `invalid_key`, expecting `md5`, `sha1`, or `sha256`
    1 | require 'ns.mod.li' { invalid_key: '' }
      |                       ^^^^^^^^^^^
